### ISSUE TITLE
_getValue: function _getValue() { throws error

### ISSUE THREAD

**Issue by vinodhrajamuralidharan** [thefrontside/emberx-select#163]
### Ember Version / Ember CLI Version:

**Ember Version:*~2.7.0* 
**Ember CLI Version:*^2.7.0*

### Expected Behavior

I have 3 select boxes. The concept is like continent - country - county

When I select continent value two times it throws error in x-select.js  on  '_getValue: function _getValue() ' -- like 'Cannot read property 'is' of undefined'

(Describe how things should work.)

### Actual Behavior

It should not throw error

### Steps To Reproduce:
1. selecting a value first time
2. selecting a different value immediately
3. throw error
<img width="1673" alt="screen shot 2016-11-18 at 14 20 08" src="https://cloud.githubusercontent.com/assets/8244992/20433319/eea4b4c8-ad9a-11e6-8e56-cfeb196117e0.png">

### Ember Twiddle / Example repo / Failing test:

_Please create a reproduction of your [bug with this ember-twiddle](https://ember-twiddle.com/278fcd6b13e8b9a06d09e7023f178496). This is extremely helpful and speeds up the debugging process._

**Comment by Robdel12** [comment]
Hey @vinodhrajamuralidharan 

Can you give master a shot and see if that fixes your issues? There are a lot of changes on master since it's basically x-select 3.0 but I think this has been fixed

**Comment by vinodhrajamuralidharan** [comment]
3.0.0 is not available in the branch? How I can, mention this in the
package JSON?


Regards,
VinodhRaja.M

**Comment by Robdel12** [comment]
Instead of a version number you would use the github repo instead:

```json
// package.json
"emberx-select": "thefrontside/emberx-select#master",
```

**Comment by vinodhrajamuralidharan** [comment]
When I take this branch, I get  the following error...
<img width="953" alt="screen shot 2016-11-19 at 12 09 01" src="https://cloud.githubusercontent.com/assets/8244992/20455318/0a18401a-ae51-11e6-8949-b4bf54f91b81.png">

**Comment by Robdel12** [comment]
should be closed with #164 and released with v2.2.3
